### PR TITLE
Created the FAQ and Donate Page

### PR DESCRIPTION
**Pull request by Lucasha11**
Created both the FAQ and Donate pages

### CODE REVIEW

**Comment by ShawnAviles** [comment]
@Lucasha11 pls put photo of the pages that were added - would make reviewing PR a lot easier so we know what we are looking for

**Comment by ShawnAviles** [comment]
There is only a jpg file added in this PR... no markdown so I assume it's not done or you may have not committed the pages yet

**Comment by Lucasha11** [comment]
FAQ Page:
Screenshot 2024-12-11 at 1 03 08 PM
Screenshot 2024-12-11 at 1 03 30 PM
Screenshot 2024-12-11 at 1 03 44 PM
Screenshot 2024-12-11 at 1 03 57 PM

Donate Page: 
Screenshot 2024-12-11 at 1 05 23 PM
Screenshot 2024-12-11 at 1 05 31 PM

**Comment by Lucasha11** [comment]
[Donate page](http://localhost:3000/donate)
[FAQs Page](http://localhost:3000/FAQs)

File: public/FAQ/FAQs cover.jpg
Comment (ShawnAviles) [inline public/FAQ/FAQs cover.jpg:1]: Should there be duplicates of these images saved?

File: content/pages/Donate.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - heroBackgroundImage:
+      src: /Donate/Donatepage.jpg
+      alt: Donate
+      colorFilter: '#6C6A6A'
+    heroHeaderText:
+      text: |
+        # <ColoredText text="DONATE" color="White" />
+      position: Left
+    _template: hero
+  - richTextLeft: "<YoutubeEmbed url=\"https://www.youtube.com/watch?reload=9&v=l_sm8Q1r3d8\" />\n\nYour donation matters! Here’s the impact you can make:\n\n\U0001F49A $25 = 21 more people supported\n\U0001F49A $50 = 42 more people supported\n\U0001F49A $100 = 84 more people supported\n\U0001F49A $250 = 210 more people supported\n\U0001F49A $500 = 420 more people supported\n\U0001F49A $1,000 = 840 more people supported\n\nEvery contribution ensures #SafeHelpIsInReach for our diverse LGBTQ+ community. \U0001F308\U0001F3F3️‍⚧️\n"
```
Comment (ShawnAviles) [inline content/pages/Donate.mdx]: I believe the youtube video id is incorrect. doesn't play anything

Screenshot 2024-12-17 at 11 33 56 PM

**Review by ShawnAviles — CHANGES_REQUESTED** [review]
There were a lot of changes made to components in the PR, was that intentional? or is that some issue from not pulling from main and other commits got added to this? @miguel-merlin @Lucasha11

**Comment by ShawnAviles** [comment]
Also the routes are all caps, everywhere else they are all lowercase.

i.e. `/Donate` should be `/donate` and `/FAQs` should be `/faqs` for consistency

**Review by j-ferber — APPROVED** [review]
LGTM

**Review by ShawnAviles — APPROVED** [review]

**Pull request merged**